core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4196,6 +4211,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_INCLUDE_OPERATION_ID =
+      new Builder(Name.USER_FILE_INCLUDE_OPERATION_ID)
+          .setDefaultValue(true)
+          .setDescription("Whether to send a unique operation id with designated FS calls.")
```

Comment:
same comment about `fs` vs filesystem. Anyways, if you're going to keep it short, at least keep capital/lowercases consistent.